src/class-utils.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Parse.ly utilities class
+ *
+ * @package Parsely
+ * @since 3.2.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely;
+
+use WP_Post;
+
+/**
+ * Utility methods to be reused across the codebase.
+ *
+ * @since 3.2.0
+ */
+class Utils {
+	/**
+	 * Generate the URL for the link.
+	 *
+	 * @since 2.6.0
+	 * @since 3.2.0 Moved to class-utils.php. Renamed from `generate_url`.
+	 *
+	 * @param WP_Post $post   Which post object or ID to check.
+	 * @param string  $apikey API key or empty string.
+	 * @return string
+	 */
+	public static function generate_parsely_post_url( WP_Post $post, string $apikey ): string {
```

Comment:
If the containing class is more descriptive, this function name can be a bit more terse:

```suggestion
	public static function post_stats_url( WP_Post $post, string $apikey ): string {
```

...making the call signature something like: `Parsely_Dashboard_Links::post_stats_url( ...`

Reply:
This goes very tight with the comment above.